### PR TITLE
scout-engines: add wasm3 with bls12 benchmarks

### LINKED ISSUE #145: Add bls12 testing on wasm3 via fizzy-bench

### PR DESCRIPTION
**Pull request by jwasinger**
closes #145

### CODE REVIEW

**Comment by axic** [comment]
What is the status of this?

**Comment by jwasinger** [comment]
I want to merge this and leave changes to the charts for later (because it isn't clear yet if/where we want to include wasm3 bls12 charts).

File: benchmark_results_data/scout_bignum_benchmarks.csv [hunk 1]
```diff
@@ -1,4 +1,14 @@
 engine,bench_name,parse_time,exec_time
+wasm3-with-bignums,bls12-wasmsnark-synth-loop,198.70827066171864,5151.1405701083795
```
Comment (axic) [inline benchmark_results_data/scout_bignum_benchmarks.csv:2]: Right now these numbers are not really useful if the fizzy numbers are not run at the same time.

Comment (jwasinger) [reply benchmark_results_data/scout_bignum_benchmarks.csv:2]: Ah good point.  we had a Fizzy version update that isn't accounted for dataset atm.

**Pull request merged**